docs/index.bs

```diff
@@ -1567,6 +1567,8 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
         RequestMode requestMode;
         RequestDestination requestDestination;
         RunningStatus runningStatus;
+
+        sequence&lt;RouterCondition&gt; _or;
```

Comment:
I'm unsure this has to have a leading underscore. https://webidl.spec.whatwg.org/#idl-names

This underscore in the chromium implementation actually comes from the chromium Web IDL compiler, but Web IDL spec itself doesn't think `or` is a reserved keyword?

Reply:
I think it is reserved: https://webidl.spec.whatwg.org/#index-prod-UnionType has it in monospaced font, which I think means it's a terminal symbol and thus the tokenizer will pick it up as part of the grammar? I'm pretty rusty on my grammar stuff though.

But we have had to do similar escaping in the past, e.g. note the definition of `_any` in https://dom.spec.whatwg.org/#interface-AbortSignal . So this makes sense to me.

Reply:
I couldn't find the whole list, that makes sense, thanks!